docs/docs/how-tos/configure-gitops.md

```diff
@@ -0,0 +1,75 @@
+---
+id: configure-gitops
+title: Setup Nebari GitOps
+description: Setting up GitOps for Nebari
+---
+
+# Setup Nebari gitops
+
+## What is GitOps / Infrastructure as Code (IaC)?
+
+GitOps also known as Infrastructure as Code(Iac) is the practice of using git for
+managing the state of your infrastructure. This allows for easy
+auditing and discovering the current state of your system.
+
+## Setting up IaC on Github-Actions
```

Comment:
Given that the title of the document is centered aroud `gitops`, I would expect we would use `gitops` in the headers. Or else change the title to be `Setup Nebari Infrastructure as Code`